### ISSUE TITLE
High CPU usage

### ISSUE THREAD

**Issue by choongng** [davidguttman/rambly-docs#8]
CPU usage is high leading to short battery life and warm... lap. Audio is still perfect which leads me to suspect something in the renderer, maybe a frame rate limit would help?

<img width="727" alt="image" src="https://user-images.githubusercontent.com/386088/90096759-0bb31d80-dce9-11ea-9023-7c55cfa2b118.png">

**Comment by robotskirts** [comment]
<img width="946" alt="Screen Shot 2020-08-12 at 10 20 15 PM" src="https://user-images.githubusercontent.com/403636/90097198-39e52d00-dcea-11ea-9f66-cf26cd7c844f.png">

LOL 114% overacheiver

**Comment by choongng** [comment]
Yeah, I think macOS accounting is 100% per thread which makes the math a bit squishy.

**Comment by StevenKitzes** [comment]
Not sure if related, but on my (2012 era) laptop, the interface starts to chug with about 8-10 users on screen, with noticeable frame drop for large events (AfterJS) and total page freeze/hang when there are several users with a live video stream (JS.LA main event).  If targeting support for older machines, I can help diagnose this further.
